.github/workflows/heroku_deploy.yml

```diff
@@ -0,0 +1,50 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
There is no need to put comments on each line

Reply:
Since there are the basic markup for the workflow. That is why we have comments.
I will remove these